src/tag/models.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+from utils.models import DefaultModel
+
+# We ask openalex to classify a paper into concepts, which we then present as tag candidates from which users can pick zero or more to attach to the paper.
```

Comment:
Are we going to ask users to pick tags? Perhaps we shouldn't do this because openalex already did so.
I think we should just accept and persist the list of concepts - at least for V1

Reply:
in the user workflow, we'll have users pick which tags they should accept from OpenAlex. Discussed in the google doc

Reply:
Users picking tags from the concepts openalex returns is what we agreed on in the [doc](https://docs.google.com/document/d/16FDcywdc3kjERkAkA73jaLrpcJoGb30lYZYinKGpDuQ/edit), but we may need to cut it from the trial project scope. I'd love to get it in but will have a better idea on how realistic that is as we approach the presentation on 10/14.